spec/integration/put_content/new_edition_spec.rb

```diff
@@ -104,4 +110,42 @@
 
     include_examples "creates a change note"
   end
+
+  context "and the payload includes last_edited_by" do
```

Comment:
**issue:** test location

The tests look good in this third commit, but is this behaviour specific to new editions? I wonder if it belongs in the main `put_content_spec`, optionally in a `describe` block like we did for [optionally setting a content ID alias](https://github.com/alphagov/publishing-api/blob/dcdc40cdcb898b7816147d8c44225507788fb62a/spec/commands/v2/put_content_spec.rb#L107)?

Reply:
I've left the tests in the integration tests, but as a shared example for both new editions and editions with existing drafts.